bazel/repository_locations.bzl

```diff
@@ -183,10 +184,10 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/msgpack/msgpack-c/releases/download/cpp-3.2.1/msgpack-3.2.1.tar.gz"],
     ),
     com_github_google_jwt_verify = dict(
-        sha256 = "d422a6eadd4bcdd0f9b122cd843a4015f8b18aebea6e1deb004bd4d401a8ef92",
-        strip_prefix = "jwt_verify_lib-40e2cc938f4bcd059a97dc6c73f59ecfa5a71bac",
-        # 2020-02-11
-        urls = ["https://github.com/google/jwt_verify_lib/archive/40e2cc938f4bcd059a97dc6c73f59ecfa5a71bac.tar.gz"],
+        sha256 = "974826c9ba8060d9656e6c360718e3ee5eb1622ff59ed4138f8a3cd718ea7b13",
+        strip_prefix = "jwt_verify_lib-2c7ddcd871c3d902e089787e244ebe49723652cc",
+        # 2019-12-20
```

Comment:
What's the date here? Commit is dated February 11
https://github.com/Maistra/jwt_verify_lib/commit/2c7ddcd871c3d902e089787e244ebe49723652cc

Reply:
It's an old one, I have a separate PR that updates jwt lib to a more recent version.

Reply:
The comment is from maistra-1.1 branch: https://github.com/Maistra/envoy/blob/maistra-1.1/bazel/repository_locations.bzl#L180